client/src/__tests__/utils.test.js

```diff
@@ -22,4 +22,4 @@ describe('composeParams', () => {
     const expected = '?product_id=1813';
     expect(result).toBe(expected);
   });
-});
\ No newline at end of file
+});
```

Comment:
Was this just broken and missing a set of closing parens/brackets?

Reply:
this was probably one of the minor changes i made to appease the linter. like removing a blank line or something like that.